src/index.html

```diff
@@ -9,6 +9,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header-container">
+        <div class="header-container__logo">
+          <a href="#home"><img src="images/logo.png" alt="logo-moyo"></a>
+        </div>
+        <nav>
+          <ul class="menu-list">
+            <li><a class="menu-list__link is-active" href="#">Apple</a></li>
```

Comment:
[Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)
Hint 4
```suggestion
            <li>
              <a class="menu-list__link is-active" href="#">
                Apple
              </a>
            </li>
```